Makefile

```diff
@@ -16,52 +22,70 @@ else
 PACK_VER:=$(VERSION)
 endif
 
-BINDIR=$(PWD)/install
+BINDIR=$(PWD)/install-$(ENGINE)
+
+BACKENDS_PATH ?= $(BINDIR)/backends
+
+#----------------------------------------------------------------------------------------------
+
+setup:
+	@echo Setting up system...
+	$(SHOW)./deps/readies/bin/getpy
+	$(SHOW)./system-setup.py
+
+.PHONY: all setup build clean deps pack pack_ramp pack_deps test
+
+all: build
 
-.PHONY: all clean deps pack rlec_runpath_fix
+ifeq ($(DEBUG),1)
+CMAKE_FLAGS += -DCMAKE_BUILD_TYPE=Debug
```

Comment:
We probably should explicitly set `CMAKE_BUILD_TYPE` to `Release` if `DEBUG` is not specified.

Reply:
This was added ad-hoc. I would like to have the ARM stuff integrated first, as it has a better infrastructure of handling debug/release variants.